### PR TITLE
Adding note on 5.1 Support Lifecycle

### PR DESCRIPTION
**Pull request by FriedrichWeinmann**
# PR Summary

Fixes https://github.com/MicrosoftDocs/PowerShell-Docs/issues/6551.

Adding a note, clarifying that PS5.1 is supported effectively indefinitely

## PR Context
Frequent confusion about Windows PowerShell support cycle when higher numbers are going out of support.

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [x] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [ ] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

## PR Checklist

- [ ] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [ ] PR has a meaningful title
- [ ] PR is targeted at the _staging_ branch
- [ ] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [ ] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

**Comment by rjmholt** [comment]
Think @joeyaiello and possibly @SteveL-MSFT are the best people to get the wording right on this. The fundamental issue is that the question of the Windows PowerShell support lifecycle is actually a series of questions:

- Will there come a time when Windows PowerShell 5.1 is EOL?
- Where is the current support lifecycle policy of Windows PowerShell documented? There we need to explain the idea that any particular Windows PowerShell version is tethered to the OS, and is supported as long as that particular OS is supported
- If Windows PowerShell is supported for as long as the OS it ships in is, and new Windows OSes are being shipped today with Windows PowerShell 5.1, is there a prospect that new versions of Windows will not ship Windows PowerShell 5.1?

**Comment by chasewilson** [comment]
> Think @joeyaiello and possibly @SteveL-MSFT are the best people to get the wording right on this. The fundamental issue is that the question of the Windows PowerShell support lifecycle is actually a series of questions:
> 
> * Will there come a time when Windows PowerShell 5.1 is EOL?
> * Where is the current support lifecycle policy of Windows PowerShell documented? There we need to explain the idea that any particular Windows PowerShell version is tethered to the OS, and is supported as long as that particular OS is supported
> * If Windows PowerShell is supported for as long as the OS it ships in is, and new Windows OSes are being shipped today with Windows PowerShell 5.1, is there a prospect that new versions of Windows will not ship Windows PowerShell 5.1?

@joeyaiello or @SteveL-MSFT would you be able to comment on the best way to address these questions?

**Comment by joeyaiello** [comment]
I plan on reaching out to folks in support to get a Windows PowerShell page equivalent to [this .NET Framework lifecycle page](https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework-47) that includes a version of the note that they have there:

> .NET Framework is a component of the Windows OS. Components receive the same support as their parent product or platform.

To be perfectly frank, I want to be careful not to add much more additional language than that, as the support lifecycle of each version of Windows is documented very particularly, and I don't want to introduce clarification that could be inaccurate. E.g. it's our understanding today that the support is "practically indefinite", but I can't say that with 100% certainty forever. 

I'd reword it like so, and I'd actually add it to the top of the doc rather than in the middle. Thoughts on that? 

> This document is about support for PowerShell Core 6 and PowerShell 7.
> Windows PowerShell (1.0 - 5.1) is a component of the Windows OS. Components receive the same support as their parent product or platform.
> For more information, see [Product and Services Lifecycle Information](https://docs.microsoft.com/en-us/lifecycle/products/)

**Comment by chasewilson** [comment]
> I plan on reaching out to folks in support to get a Windows PowerShell page equivalent to [this .NET Framework lifecycle page](https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework-47) that includes a version of the note that they have there:
> 
> > .NET Framework is a component of the Windows OS. Components receive the same support as their parent product or platform.
> 
> To be perfectly frank, I want to be careful not to add much more additional language than that, as the support lifecycle of each version of Windows is documented very particularly, and I don't want to introduce clarification that could be inaccurate. E.g. it's our understanding today that the support is "practically indefinite", but I can't say that with 100% certainty forever.
> 
> I'd reword it like so, and I'd actually add it to the top of the doc rather than in the middle. Thoughts on that?
> 
> > This document is about support for PowerShell Core 6 and PowerShell 7.
> > Windows PowerShell (1.0 - 5.1) is a component of the Windows OS. Components receive the same support as their parent product or platform.
> > For more information, see [Product and Services Lifecycle Information](https://docs.microsoft.com/en-us/lifecycle/products/)

Thank you @joeyaiello. @FriedrichWeinmann can you make these changes to your PR? Then we will get it merged in.

**Comment by chasewilson** [comment]
@FriedrichWeinmann, I went ahead and made the change suggested by Joey, and am going to go ahead and merge it. 

Thank you again for your contribution, this is a much needed change!
